packages/hooks/src/mocks/resources.ts

```diff
@@ -1,3 +1,4 @@
+import { CLASSIFICATION_AUDIENCES, AFFILIATIONS } from './../../../lib/src/user/constants';
```

Comment:
Ah shoot.. yea, we can't import relative directories here.. it has to come in by way of the `@osu-wams/lib` package exports.